src/components/CodeInput.tsx

```diff
@@ -1,272 +1,122 @@
-                // numberOfLines is currently Android only on TextInput
-                // workaround is to set the minHeight on iOS :/
-                numberOfLines={Platform.OS === 'ios' ? undefined : numberOfLines}
-                inputStyle={{
-                  ...(shortVerificationCodesEnabled && {
-                    ...fontStyles.large,
-                    textAlign: 'center',
-                  }),
-                  minHeight:
-                    Platform.OS === 'ios' && numberOfLines
-                      ? LINE_HEIGHT * numberOfLines
-                      : undefined,
-                }}
-                autoCapitalize="none"
-                autoFocus={autoFocus}
-                testID={testID}
-              />
-            ) : (
-              <Text
-                style={shortVerificationCodesEnabled ? styles.codeValue : styles.codeValueLong}
-                numberOfLines={1}
-              >
-                {inputValue || ' '}
-              </Text>
-            )}
-          </View>
-          {showStatus && (
-            <View style={styles.statusContainer}>
-              {showSpinner && <ActivityIndicator size="small" color={colors.primary} />}
-              {showCheckmark && <Checkmark testID={testID ? `${testID}/CheckIcon` : undefined} />}
-              {showError && (
-                <InfoIcon
-                  color={colors.error}
-                  testID={testID ? `${testID}/ErrorIcon` : undefined}
-                />
-              )}
-            </View>
-          )}
-        </View>
-        {showInput && (
-          <ClipboardAwarePasteButton
-            getClipboardContent={getFreshClipboardContent}
-            shouldShow={shouldShowClipboardInternal()}
-            onPress={onInputChange}
+    <Card rounded={true} shadow={null} style={[styles.container, style]}>
+      {showStatus && <View style={styles.statusContainer} />}
+      <View style={styles.innerContent}>
+        {showInput ? (
+          <TextInput
+            textContentType={'oneTimeCode'}
+            showClearButton={false}
+            value={inputValue}
+            placeholder={inputPlaceholder}
+            onChangeText={onInputChange}
+            // This disables keyboard suggestions on iOS, but unfortunately NOT on Android
+            // Though `InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS` is correctly set on the native input,
+            // most Android keyboards ignore it :/
+            autoCorrect={false}
+            // On Android, the only known hack for now to disable keyboard suggestions
+            // is to set the keyboard type to 'visible-password' which sets `InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD`
+            // on the native input. Though it doesn't work in all cases (see https://stackoverflow.com/a/33227237/158525)
+            // and has the unfortunate drawback of breaking multiline autosize.
+            // We use numberOfLines to workaround this last problem.
```

Comment:
is this comment still valid? Looks like we force keyboardType to `number-pad`, maybe can just be inlined below instead of a variable?